packages/xstate-immer/package.json

```diff
@@ -38,11 +38,11 @@
   },
   "dependencies": {},
   "peerDependencies": {
-    "immer": "^5.0.0",
+    "immer": "^6.0.3",
```

Comment:
is 6.0.3 required? or could we lower this to 6.0.0?

Reply:
Might as well use the latest 😛

Reply:
👍